data/tlg0064/tlg001/tlg0064.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,197 @@
+                <sponsor>HARVARD COLLEGE LIBRARY</sponsor>
+                <funder/>
+                <principal>Gregory Crane</principal>
+                <meeting/>
+                <respStmt>
+                    <persName xml:id="DDD">Digital Divide Data</persName>
+                    <resp>Corrected and encoded the text</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>RHEA LESAGE</persName>
+                    <resp>Librarian for Hellenic Studies and Coordinator for the Classics Collection
+                        Development</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg0064.tlg001.opp-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2016</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United State</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+
+                            <title>Geographi Graeci Minores</title>
+                            <editor>
+                                <persName>
+                                    <name xml:lang="lat">Carolus Müller</name>
+                                    <addName xml:lang="de">Karl Müller</addName>
+                                </persName>
+                            </editor>
+                            <author ref="http://data.perseus.org/catalog/urn:cts:greekLit:tlg0064">Hanno</author>
+
+                            <imprint>
+                                <publisher>Ambroise-Firmin Didot</publisher>
+                                <pubPlace>Paris</pubPlace>
+                                <date>MDCCCLV</date>
+                            </imprint>
+
+                            <biblScope unit="volume">1</biblScope>
+
+                        </monogr>
+                        <ref target="https://archive.org/details/geographigraeci00philgoog"/>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+
+        <encodingDesc>
+
+            <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+            <p>The following text is encoded in accordance with EpiDoc standards and with the
+                CTS/CITE Architecture.</p>
+
+
+            <refsDecl n="CTS">
+
+                <cRefPattern n="paragraph" matchPattern="(.+).(.+)"
```

Comment:
Since there is only one level of citations, the match pattern should be "(.+)". When you have more than one, then add more of these separated by periods.